### PR TITLE
fix(UploadDialog): show/hide file extension on input blur/focus

### PR DESCRIPTION
**Pull request by suprko**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/1894/desktop-file-extension-is-not-shown-in-upload-confirmation-dialog
 
#### Testing instructions  
In upload-to-chat dialog:
* file extension should hide when input is focused
* file extension should show when input is not focused
  * except, input should remain blank when file name is left blank
* uploaded file should end up with proper file name (name + extension)

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: src/ui/shared-components/UploadDialog.jsx [hunk 1]
```diff
@@ -52,6 +53,23 @@ class UploadDialog extends React.Component {
         return fileHelpers.getFileIconType(this.fileExt);
     }
 
+    // "Display" filename can be different from true file name (ext shown on input blur, hidden on focus)
+    @observable fileNameToDisplay = '';
```
Comment (anri-asaturov) [inline src/ui/shared-components/UploadDialog.jsx]: Replace all this and below with one computed `displayFileName` (or whatever name you like)
computed should be based on 'isFocused' observable that changes on blur/focus

Comment (suprko) [reply src/ui/shared-components/UploadDialog.jsx]: updated

**Comment by anri-asaturov** [comment]
ready to test

**Comment by ebilg** [comment]
ready to merge

**Pull request merged**